README.md

```diff
@@ -1,26 +1,26 @@
+App Studio is an effort to automate build and delivery of Red Hat products. With App Studio, you have access to single, simple workflow for developing, testing, and releasing containerized applications while ensuring compliance with enterprise security standards.
 
-## Stonesoup documentation links
+## App Studio documentation links
 
-* [Getting-Started Guide](https://red-hat-stone-soup.pages.redhat.com/stonesoup-documentation/users/getting_started.html)
-* [User Documentation](https://red-hat-stone-soup.pages.redhat.com/stonesoup-documentation/users/index.html/120v38DZY6iuHlnyJJ2_k78vwhNiA38LqIZIzTT9j8aM)
-* [Developer Documentation](https://red-hat-stone-soup.pages.redhat.com/stonesoup-documentation/developers/index.html)
-* [Schema Documentation](https://red-hat-stone-soup.pages.redhat.com/stonesoup-documentation/schema/index.html)
-* [Architecture Overview](https://red-hat-stone-soup.pages.redhat.com/stonesoup-documentation/architecture/index.html)
+* [Getting started guide](https://redhat-appstudio.github.io/docs.appstudio.io/Documentation/main/getting-started/get-started)
+* [User documentation](https://redhat-appstudio.github.io/docs.appstudio.io/Documentation/main)
+* [Upstream landing page](https://redhat-appstudio.github.io/appstudio.docs.ui.io/)
+<!-- * [Schema Documentation](https://red-hat-stone-soup.pages.redhat.com/stonesoup-documentation/schema/index.html) -->
+<!-- * [Architecture Overview](https://red-hat-stone-soup.pages.redhat.com/stonesoup-documentation/architecture/index.html) -->
 
-## External Links
+## External links
 
-* [Stonesoup Console (Confluence)](https://docs.engineering.redhat.com/pages/viewpage.action?pageId=256849149)
-* [Stonesoup Project Status Dashboard](https://docs.google.com/document/d/1wzJu-wOYez5p875kl0QkgQ6b2i9x_T9983YdxTxBd-I/edit?usp=sharing)
+* [HACDOCS project in JIRA](https://issues.redhat.com/projects/HACDOCS/summary)
+* [HACBS hub on Confluence](https://docs.engineering.redhat.com/pages/viewpage.action?pageId=256849149)
 
-## Stonesoup Contact Information
+## App Studio contact information
 
-* [Stonesoup Documentation Home Page](https://red-hat-stone-soup.pages.redhat.com/stonesoup-documentation/)
-* [Stonesoup Documentation GitLab Repository](https://gitlab.cee.redhat.com/red-hat-hybrid-application-cloud-build-services-documentation/Stonesoup-documentation)
-* [Email: Stonesoup Development Team](mailto:Stonesoup@googlegroups.com)
-* [Google chat: Stonesoup Users](https://groups.google.com/g/Stonesoup)
+* [App Studio documentation home page](https://redhat-appstudio.github.io/appstudio.docs.ui.io/)
+* [App Studio documentation GitLab repository](https://gitlab.cee.redhat.com/red-hat-trusted-application-pipeline/red-hat-trusted-application-pipeline)
```

Comment:
I think we can update line # 21 line to convey info about the GitHub repo instead of GitLab. We no longer maintain the GitLab repo.

Reply:
Thanks for letting me know, I'll remove GitLab info then.

Reply:
Since this is a readme for out github repo, let's not include a link to this repo here. I'll just remove a gitlab one then. And since docs home page is linked in a section above, it looks like we can remove this Contact info section altogether.